multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/utils/MapperAuthUtil.java

```diff
@@ -6,52 +6,51 @@
 
 package com.sngular.api.generator.plugin.openapi.utils;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
+import static com.sngular.api.generator.plugin.commonApi.MapperUtilCommon.getTextValue;
 
+import java.util.*;
```

Comment:
Here you have another * on a import, so I tell you the same as my previous comment.